.vscode/settings.json

```diff
@@ -8,7 +8,35 @@
     "**/lib-es6": true,
     "target": true
   },
-  "prettier.semi": false,
-  "prettier.singleQuote": true,
-  "prettier.trailingComma": "es5"
+  "editor.formatOnSave": true,
```

Comment:
You mean `editor.formatOnSave` one? Or `prettier` config moved to `eslint`? 
We can remove `formatOnSave` if you don't like it. The format will be fixed by `lint-staged` when the code is committing anyway.

Reply:
Sorry, I should provide more context here.

All the editor settings that you're adding will change the developer's IDE normal behavior and I usually tend to avoid doing this as everyone has their preferences.

What we can do and we already are doing — add some pre-commit hooks to ensure code passes eslint and prettier.

And it's not fair to add settings for vscode, and completely ignore WebStorm, for example. Having settings for all IDE will just pollute the project.

What do you think?